src/Persistent/Implementation/AcidState.hs

```diff
@@ -66,8 +66,8 @@ instance MonadIO Persist where
 askDbM :: Query AulaData AulaData
 askDbM = ask
 
-getDbM :: DbField a -> Update AulaData a
-getDbM l = use (dbFieldTraversal l)
+getDbM :: Update AulaData AulaData
+getDbM = get
```

Comment:
Yes, you are right, I went overboard with that; indeed, getDbM doesn't need DbField.